cmd/destroy/destroy.go

```diff
@@ -191,6 +191,15 @@ func Destroy(ctx context.Context) *cobra.Command {
 			os.Setenv("KUBECONFIG", kubeconfigPath)
 			defer os.Remove(kubeconfigPath)
 
+			if options.Name == "" {
```

Comment:
We should also have this logic in the deploy.
We can use the same repo switching the deploy with the destroy sections and running `okteto deploy`

Reply:
I think deploy command will not face this issue because there has been nothing prior to the deploy that has created resources with a different name (as is the case with the destroy)